### DISCUSSION TITLE
Consulta: Envio de Factura Electronica a cliente

### DISCUSSION THREAD

**Discussion by kakarata** [OdooAR/odoo-argentina-doc#15] [Q&A] [answered]
Buenas, uso lo siguiente:
Versión de Odoo: 14.0 CE
Sistema Operativo: debian 11
Desarrollador de la Localización: Codize
Consulta:
Al usar l10n_ar_afipws_fe, encuentro que solo veo el pdf con formato electronico (QR includo) cuando sobre una factura voy al boton Imprimir, Facturas Electronicas. Pero no asi la que manda Odoo al cliente por mail y puede ver el cliente en el portal de usuario, que es un formato estilo LibreOffice.
Es correcto que trabaje asi? Si asi es, esta pensado el envio automatico al cliente de alguna forma?

Gracias!

**Comment by kakarata** [comment]
Existe de casualidad alguna documentación para ver como funciona y las capacidades? O tocaria leer el codigo unicamente?
Gracias

**Comment by ibuioli** [comment, marked as answer]
Buenas, en el caso de la localización de Codize se optó porque la factura electrónica sea un reporte distinto, más que nada para aquellos que utilizan la factura tradicional de Odoo en lugar de la electrónica, o que alternan su uso. Para hacer que en el mail se mande la electrónica en lugar de la regular, es una configuración de la plantilla de correo. En modo Desarrollador te dirigís a Ajustes > Tecnico > Plantillas (dentro de la sección Correo Electrónico) y buscás la plantilla que se manda desde el registro de la factura por mail, que es la que se llama Invoice: Send by email. Dentro de ese registro buscás la pestaña Configuración avanzada y editas el campo Informe opcional a generar y adjuntar que va a decir "Facturas" para que diga "Facturas electrónicas". 
Despues para la preview en el portal, si toca cambiar algunas cosas del código de Odoo para que traiga el reporte de la FE en lugar de la factura normal, voy a tratar de dejarlo por acá mas adelante (tengo que ubicar el código, pero como pista te puedo decir que es parte del core de odoo eso, dentro de todo lo son los módulos de portal). Lo que me llama la atención es lo del LibreOffice, la localización de Codize no utiliza reportes Aeroo (que si usaba la vieja de AdHoc), no se si tendrás instalado alguna otra cosa. Saludos!
